Gemfile

```diff
@@ -25,6 +25,8 @@ end
 group :development, :test do
   gem 'byebug', '~> 10'
   gem 'foreman', '~> 0.87.0'
-  gem 'rspec-rails', '~> 3'
+  gem 'rspec-rails', '~> 4.0.0.beta2'
```

Comment:
This was necessary due to a known bug in the gem: https://github.com/rspec/rspec-rails/issues/2177